client/packages/common/src/intl/locales/pt/common.json

```diff
@@ -1032,5 +1032,72 @@
   "title.stock-on-hand": "Estoque Disponível",
   "tools": "Ferramentas",
   "warning.caps-lock": "Advertência: Caps Lock está ativada",
-  "warning.nothing-to-supply": "Nada para fornecer!"
+  "warning.nothing-to-supply": "Nada para fornecer!",
```

Comment:
Ah sorry, ignore these, I think I pulled a more recent version of develop into my branch than the base of this PR 👀